Storage.md

```diff
@@ -0,0 +1,209 @@
+This guide will describe how FullScreenShenanigans projects keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Items](#items)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an HTML container to show these values.
+
+## Local Storage
+
+ItemsHoldr keeps track of items and their values while the game is running.
+It can also save this information to `localStorage`.
+Other websites also use `localStorage`, so ItemsHoldr has a prefix property to differentiate which information belongs to which source.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    prefix: "FullScreenShenanigans"
+});
+```
+
+## Items
+
+To add, update, and remove an item in the collection, use `addItem`, `setItem`, and `removeItem` respectively.
+
+```typescript
+ItemsHolder.addItem("color", { value: "red" });
+ItemsHolder.setItem("color", "purple");
+ItemsHolder.removeItem("color");
+```
+
+To manually update a changed item's value in localStorage, use `saveItem`.
+To save all the items' values to localStorage, use `saveAll`.
+
+```typescript
+ItemsHolder.saveItem("color");
+ItemsHolder.saveAll();
+```
+
+`increase` and `decrease` add and subtract respectively from the item's value.
+`toggle` flips the boolean value of the item.
+
+```typescript
+ItemsHolder.increase("weight", 14);
+ItemsHolder.decrease("weight", 10);
+ItemsHolder.toggle("started");
+```
+
+`checkExistence` can be used to check if there is an item under the specified key and if not, add it to the collection.
+
+```typescript
+ItemsHolder.checkExistence("color");
+```
+
+By default if an item does not exist when `setItem`, `getItem`, `increase`, or `toggle` are called, the item is then added to the collection.
+Define `allowNewItems` to explicitly enable or disable this feature.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ allowNewItems: false });
```

Comment:
You put the contents of the settings object on the next line above. Can you do it here as well?